src/system_database.ts

```diff
@@ -654,6 +671,23 @@ export class PostgresSystemDatabase implements SystemDatabase {
     }
   }
 
+  async recordChildWorkflow(
+    parentUUID: string,
+    childUUID: string,
+    functionID: number,
+    functionName: string,
+  ): Promise<void> {
+    try {
+      await this.pool.query<operation_outputs>(
+        `INSERT INTO ${DBOSExecutor.systemDBSchemaName}.operation_outputs (workflow_uuid, function_id, function_name, child_id) VALUES ($1, $2, $3, $4) ON CONFLICT DO NOTHING;`,
```

Comment:
```suggestion
        `INSERT INTO ${DBOSExecutor.systemDBSchemaName}.operation_outputs (workflow_uuid, function_id, function_name, child_id) VALUES ($1, $2, $3, $4);`,
```
I don't think we use `ON CONFLICT DO NOTHING;`, instead, check the error the way operation output does:
```javascript
if (err.code === '40001' || err.code === '23505') {
    // Serialization and primary key conflict (Postgres).
    throw new DBOSWorkflowConflictUUIDError(workflowUUID);
  } else {
    throw err;
  }
```
Otherwise, it may cause race conditions where this row is updated twice.